frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -40,6 +145,34 @@ function ReaderDictionary:addToMainMenu(menu_items)
         text = _("Dictionary settings"),
         sub_item_table = {
             {
+                -- text = _("Installed dictionaries"),
+                text_func = function()
+                    local nb_available = #available_ifos
+                    local nb_disabled = 0
+                    for _ in pairs(G_reader_settings:readSetting("dicts_disabled")) do
+                        nb_disabled = nb_disabled + 1
+                    end
+                    local nb_str = nb_available
+                    if nb_disabled > 0 then
+                        local nb_enabled = nb_available - nb_disabled
+                        nb_str = nb_enabled .. "/" .. nb_available
+                    end
+                    return T(_("Installed dictionaries (%1)"), nb_str)
+                end,
+                sub_item_table = self:genDictionariesMenu(),
+            },
+            {
+                text = _("Info on dictionaries ordering"),
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = T(_("If you'd like to change the order in which dictionaries are queried (and their results displayed), you can:\n"..
```

Comment:
D'oh, I totally overlooked this. See #3151. Currently it needs to be either one long string or use `[[]]` for multiline. I'd like the translation extraction script to do concatenation (which it did where it came from) but in any case I don't have time to look at it today. :-)